.github/workflows/publish-docs.yml

```diff
@@ -0,0 +1,19 @@
+name: Publish Docs via GitHub Pages
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  build:
+    name: Deploy Docs
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout master
+        uses: actions/checkout@v1
```

Comment:
It looks like v2 is the latest for this. Any particular reason we are using v1?

See: 
https://github.com/actions/checkout
https://github.com/actions/checkout/blob/master/CHANGELOG.md